.github/CONTRIBUTING.md

```diff
@@ -418,26 +418,26 @@ for(var/atom/thing as anything in bag_of_atoms)
 ### SQL
 * Do not use the shorthand sql insert format (where no column names are specified) because it unnecessarily breaks all queries on minor column changes and prevents using these tables for tracking outside related info such as in a connected site/forum.
 
-* All changes to the database's layout(schema) must be specified in the database changelog in SQL, as well as reflected in the schema files
+* All changes to the database's layout(schema) should be specified in the database changelog in SQL, as well as reflected in the schema files
 
-* Any time the schema is changed the `schema_revision` table and `DB_MAJOR_VERSION` or `DB_MINOR_VERSION` defines must be incremented.
+* Any time the schema is changed the `schema_revision` table and `DB_MAJOR_VERSION` or `DB_MINOR_VERSION` defines should be incremented.
 
-* Queries must never specify the database, be it in code, or in text files in the repo.
+* Queries should never specify the database, be it in code, or in text files in the repo.
 
-* Primary keys are inherently immutable and you must never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
+* Primary keys are inherently immutable and you should never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
 
-* The ttl for data from the database is 10 seconds. You must have a compelling reason to store and reuse data for longer then this.
+* The ttl for data from the database is 10 seconds. You should have a compelling reason to store and reuse data for longer then this.
```

Comment:
I don't know what this means but it's probably best not to water down code standards without a good justification

Reply:
While this entire pr was made from a 5 second find/replace, this line here I added in, and I support moving to should.